tests/array_manipulation_tests.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+from base_test import ArkoudaTest
+from context import arkouda as ak
+import pytest
+
+
```

Comment:
Can you add the tests to PROTO_tests/tests instead?  We are trying to deprecate the tests folder.